src/components/NcAppNavigation/NcAppNavigation.vue

```diff
@@ -183,6 +193,13 @@ export default {
 		gap: var(--default-grid-baseline, 4px);
 		padding: calc(var(--default-grid-baseline, 4px) * 2);
 	}
+	.app-navigation__content {
+		height: 100%;
+		display: flex;
+		flex-direction: column;
+		flex-grow: 0;
+		flex-shrink: 0;
```

Comment:
These styles are moved from `.app-navigation`, which previously was the container of slots and the list, to the new container. Could we now remove them from the `.app-navigation` or they are still required?

Reply:
it is hard to test all apps and the risk to break something is high. I could remove https://github.com/nextcloud/nextcloud-vue/blob/master/src/components/NcAppNavigation/NcAppNavigation.vue#L161-L162 but would like leave it to be safe.

Reply:
done